src/recipe.js

```diff
@@ -7,10 +9,18 @@ class Recipe {
     this.ingredients = recipe.ingredients;
   }
   calculateIngredientsCost() {
-    // return this.ingredients.map(i => {
-    //   ingredientData.find(ingredient => ingredient === i);
-    // });
+    const costInCents = this.ingredients.reduce((totalCost, recipeIngredient) => {
+      ingredientsData.forEach(ingredient => {
+        if (ingredient.id === recipeIngredient.id) {
+          let ingredientTotal = ingredient.estimatedCostInCents * recipeIngredient.quantity.amount;
+          totalCost += ingredientTotal;
+        }
+      })
+      return totalCost;
+    }, 0);
+
+    return `$${costInCents.toString().slice(0, 1)}.${costInCents.toString().slice(1, 3)}`
```

Comment:
This all looks really solid. I do think there is a built in method that allows you to convert numbers to the correct currency syntax. .toLocaleString()